docs/DEVELOPER_GUIDE.md

```diff
@@ -193,12 +186,36 @@ this:
     └── kubectl-foo-windows.exe
 ```
 
-The resulting installation directory would up just with:
-
-```text
-.
-└── krew-foo-windows.exe
-```
+- To copy all files in the `bin/` directory of the extracted archive to
+  the root of your plugin, use the default:
```

Comment:
hmm, to use the default, maybe they shouldn't need to specify `files:`?

Reply:
Well, my thinking was to keep the two examples as similar as possible. But I agree that this looks odd. I changed the wording.